environments-ui/src/Environments.js

```diff
@@ -0,0 +1,176 @@
+                                        />
+                                    </ListItemSecondaryAction>
+                                </ListItem>
+                            );
+                        })}
+                    </List>
+                </div>
+            </Drawer>
+            <main className={classes.content}>
+                <Toolbar />
+                <Grid container spacing={4}>{
+                    environments.map(function (envs, index) {
+                        const env = envs[0]
+                        const value = env.name.split(' ')[0]
+                        if (checked.indexOf('all') > -1 || checked.indexOf(value) > -1) {
+                            return (
+                                <Grid item key={index}>
+                                    <Card className={classes.root}>
+                                        <CardActionArea href={env.readme}>
+                                            <CardMedia
+                                                style={{
+                                                    objectFit: 'fill',
+                                                    height: '100px',
+                                                    paddingLeft: '5px',
+                                                    paddingRight: '5px',
+                                                    paddingTop: '5px'
+                                                }}
+                                                component="img"
+                                                alt="Contemplative Reptile"
+                                                height="140"
+                                                image={env.icon}
+                                                title="Contemplative Reptile"
+                                            />
+                                            <CardContent>
+                                                <Typography variant="body1" component="h2" >
+                                                    {env.name}
+                                                </Typography>
+                                                <Typography variant="body2" color="textSecondary" component="p">
+                                                    {env.shortDescription}
+                                                </Typography>
+                                            </CardContent>
+                                        </CardActionArea>
+                                        {/* <CardActions>
+                    <Button size="small" color="primary">
+                      Share
+        </Button>
+                    <Button size="small" color="primary">
+                      Learn More
+        </Button>
+                  </CardActions> */}
+                                    </Card>
+                                </Grid>
+                            )
+                        }
+                    })
+                }
```

Comment:
If this is not needed, can you please remove it?

Reply:
removed